openvpn/client.go

```diff
@@ -86,6 +85,9 @@ func (client *openVpnClient) Start() error {
 		return err
 	}
 
+	addr := client.management.ActiveSocketAddress()
```

Comment:
We don't need directoryRuntime anymore as a parameter? In that case it can be removed